installing/installing-troubleshooting.adoc

```diff
@@ -7,7 +7,12 @@ include::_attributes/common-attributes.adoc[]
 toc::[]
 
 To assist in troubleshooting a failed {product-title} installation, you can
-gather logs from the bootstrap and control plane, or master, machines. You can also get debug information from the installation program.
+gather logs from the bootstrap and control plane machines. You can also get debug information from the installation program. If you are unable to resolve the issue using the logs and debug information, see xref:../support/troubleshooting/troubleshooting-installations.adoc#determining-where-installation-issues-occur_troubleshooting-installations[Determining where installation issues occur] for component-specific troubleshooting.
```

Comment:
Not introduced by this PR, but directly related to it: can we join these two lines into one for consistency?

Otherwise, I love the additional information provided by this PR, thank you!

Reply:
Fixed, thanks